src/commands/save.json

```diff
@@ -1,7 +1,7 @@
 {
     "SAVE": {
         "summary": "Synchronously save the dataset to disk",
-        "complexity": "PATCH__TBD__51__",
+        "complexity": "NA",
```

Comment:
BGREWRITEAOF was O(1)

Reply:
yes but this is foreground

Reply:
ohh, so it's the same as FLUSHALL.
what did we write there?

Reply:
```
O(N) where N is the total number of keys in all databases
```

not so accurate but ok